kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java

```diff
@@ -202,10 +220,111 @@ public void onChanged(Order value) {
         assertNull(orderRepository.getOpenOrder().getValue());
         verify(offerRepository).setPendingOfferByID(null);
 
+        verify(eventLogger, never()).send(any(SpendOrderCompleted.class));
```

Comment:
why do we need this "never" test?

Reply:
i want to check that no spend even it called, this is earn submission.